### ISSUE TITLE
Tests failing on 2.4.1 on Alpine Linux

### ISSUE THREAD

**Issue by ddevault** [Kozea/CairoSVG#247]
```
==================================================================================== test session starts =====================================================================================
platform linux -- Python 3.7.4, pytest-4.5.0, py-1.8.0, pluggy-0.9.0
rootdir: /home/sircmpwn/sources/aports/testing/py3-cairosvg/src/CairoSVG-2.4.1, inifile: setup.cfg
plugins: isort-0.3.1, flake8-1.0.4, celery-4.3.0
collected 43 items

setup.py ..                                                                                                                                                                            [  4%]
cairosvg/__init__.py ..                                                                                                                                                                [  9%]
cairosvg/__main__.py ..                                                                                                                                                                [ 13%]
cairosvg/bounding_box.py ..                                                                                                                                                            [ 18%]
cairosvg/colors.py ..                                                                                                                                                                  [ 23%]
cairosvg/css.py ..                                                                                                                                                                     [ 27%]
cairosvg/defs.py ..                                                                                                                                                                    [ 32%]
cairosvg/features.py ..                                                                                                                                                                [ 37%]
cairosvg/helpers.py ..                                                                                                                                                                 [ 41%]
cairosvg/image.py ..                                                                                                                                                                   [ 46%]
cairosvg/parser.py ..                                                                                                                                                                  [ 51%]
cairosvg/path.py ..                                                                                                                                                                    [ 55%]
cairosvg/shapes.py ..                                                                                                                                                                  [ 60%]
cairosvg/surface.py ..                                                                                                                                                                 [ 65%]
cairosvg/svg.py ..                                                                                                                                                                     [ 69%]
cairosvg/test_api.py ........F                                                                                                                                                         [ 90%]
cairosvg/text.py ..                                                                                                                                                                    [ 95%]
cairosvg/url.py ..                                                                                                                                                                     [100%]

========================================================================================== FAILURES ==========================================================================================
________________________________________________________________________________________ test_script _________________________________________________________________________________________

    def test_script():
        """Test the ``cairosvg`` script and the ``main`` function."""
        expected_png = svg2png(SVG_SAMPLE)
        expected_pdf = svg2pdf(SVG_SAMPLE)

        def test_main(args, exit_=False, input_=None):
            """Test main called with given ``args``.

            If ``exit_`` is ``True``, check that ``SystemExit`` is raised. We then
            assume that the program output is an unicode string.

            If ``input_`` is given, use this stream as input stream.

            """
            sys.argv = ['cairosvg'] + args
            old_stdin, old_stdout = sys.stdin, sys.stdout

            output_buffer = io.BytesIO()
            sys.stdout = io.TextIOWrapper(output_buffer)

            if input_:
                sys.stdin = open(input_, 'rb')
                sys.stdin.buffer = sys.stdin

            if exit_:
                try:
                    main()
                except SystemExit:
                    pass
                else:  # pragma: no cover
                    raise Exception('CairoSVG did not exit')
            else:
                main()

            sys.stdout.flush()
            output = output_buffer.getvalue()
            sys.stdin, sys.stdout = old_stdin, old_stdout
            return output

        with tempfile.NamedTemporaryFile(delete=False) as file_object:
            file_object.write(SVG_SAMPLE)
            file_object.flush()
            svg_filename = file_object.name
            file_object.close()

            assert test_main(['--help'], exit_=True).startswith(b'usage: ')
            assert test_main(['--version'], exit_=True).strip() == (
                VERSION.encode('ascii'))

            assert test_main([svg_filename]) == expected_pdf
            assert test_main([svg_filename, '-d', '96', '-f', 'pdf']) == (
                expected_pdf)
            assert test_main([svg_filename, '-f', 'png']) == expected_png
            assert test_main(['-'], input_=svg_filename) == expected_pdf

            # Test DPI
            output = test_main([svg_filename, '-d', '10', '-f', 'png'])
            image = cairo.ImageSurface.create_from_png(io.BytesIO(output))
            print(image.get_width(), image.get_height())
            assert image.get_width() == 40
            assert image.get_height() == 50

            temp = tempfile.mkdtemp()
            try:
                temp_1 = os.path.join(temp, 'result_1')
                # Default to PDF
                assert not test_main([svg_filename, '-o', temp_1])
>               assert read_file(temp_1) == expected_pdf
E               assert b"%PDF-1.5\n%...n738\n%%EOF\n" == b"%PDF-1.5\n%\...n738\n%%EOF\n"
E                 At index 667 diff: 50 != 49
E                 Use -v to get the full diff

cairosvg/test_api.py:212: AssertionError
------------------------------------------------------------------------------------ Captured stdout call ------------------------------------------------------------------------------------
40 50
============================================================================ 1 failed, 42 passed in 4.53 seconds =============================================================================
```

**Comment by liZe** [comment]
Thanks for the report.

Could you please attach the `result_1` PDF file?

**Comment by ddevault** [comment]
I'm not sure where it gets written to.

**Comment by ddevault** [comment]
Ah, found it.
[result_1.pdf](https://github.com/Kozea/CairoSVG/files/3529392/result_1.pdf)

**Comment by liZe** [comment]
Oh.

The generated PDF contain the creation date, and we compare 2 documents that may have not been generated in the same second. It almost always work for "fast" computers, but may fail with slower ones. I don't know how I could have missed that.

**Comment by liZe** [comment]
Fixed by 1b9542f.

**Comment by ddevault** [comment]
Thanks!
